vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -43,7 +86,7 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 	if err != nil {
 		err = fmt.Errorf(title+" - Failed conversion of id %s to pointer %s", id, err)
 		logrus.Error(err)
-		return 0, err
+		return
```

Comment:
sorry wrong line, but that's for the return above

for this one it's https://github.com/docker/docker/pull/14758/files#diff-32aab1839954fefce8574a22d2ddb66bR85